### PR TITLE
added some missing yang references for router-bgp

### PR DESCRIPTION
**Pull request by summed**
How about adding this? - Not sure what more is needed for this to be a eligible for a merge...

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,8 @@
 module github.com/CiscoDevNet/terraform-provider-iosxr
 
-go 1.20
+go 1.21
```
Comment (danischm) [inline go.mod:3]: Is there a specific reason you are bumping the go version?

**Comment by danischm** [comment]
Many thanks for opening this PR! Did you have a chance to run the corresponding acceptance tests with your changes?

**Comment by summed** [comment]
@danischm 
Some pass, while most seem to fail. Looks like some kind of timeout. One of many is here...

```
=== RUN   TestAccIosxrL2VPNPWClass
    resource_iosxr_l2vpn_pw_class_test.go:56: Step 1/3 error: Error running pre-apply refresh: exit status 1

        Error: Invalid provider configuration

        Provider "registry.terraform.io/hashicorp/iosxr" requires explicit
        configuration. Add a provider block to the root module and configure the
        provider's required arguments as described in the provider documentation.


        Error: Unable to create gNMI client

          with provider["registry.terraform.io/hashicorp/iosxr"],
          on <empty> line 0:
          (source code not available)

        Unable to create gNMI client:

        10.201.248.15:57400: context deadline exceeded
--- FAIL: TestAccIosxrL2VPNPWClass (10.50s)
```

**Comment by danischm** [comment]
Ok, in that case let's revert the change with the go version. I would not care about the other tests but only run the ones where you have modified the definition/code, for example:

```
go test -v ./... -run RouterBGPNeighborAddressFamily$
```

File: go.mod (hunk 1 above)
Comment (summed) [reply go.mod]: reverted to v0.3.2 commit

**Comment by shebang42** [comment]
Can this PR be merged?  I also need these updated BGP resources.

**Review by danischm — APPROVED** [review]

**Pull request merged**